security/Member.php

```diff
@@ -1801,22 +1801,31 @@ public function __construct() {
  * Member Validator
  *
  * Custom validation for the Member object can be achieved either through an
- * {@link DataExtension} on the Member object or, by specifying a subclass of
+ * {@link DataExtension} on the Member_Validator object or, by specifying a subclass of
  * {@link Member_Validator} through the {@link Injector} API.
  *
+ * Custom required fields can be set via config API, eg.
+ * Member_Validator:
+ *   customRequired:
+ *     - Surname
+ *
  * {@see Member::getValidator()}
  *
  * @package framework
  * @subpackage security
  */
-class Member_Validator extends RequiredFields {
-
-	protected $customRequired = array(
+class Member_Validator extends RequiredFields
+{
+	/**
+	 * Fields that are required by this validator
+	 * @config
+	 * @var array
+	 */
+	private static $customRequired = array(
```

Comment:
Oh, I just realised that changing the customRequired to config would be an API change. this COULD go into 3 branch, but you'd need to keep it as protected var to merge it into 3.3.